include/ddc/kernels/splines/matrix_sparse.hpp

```diff
@@ -121,7 +145,13 @@ class Matrix_Sparse : public Matrix
     unsigned int m_preconditionner_max_block_size; // Maximum size of Jacobi-block preconditionner
 
 public:
-    // Constructor
+    /**
+     * @brief Matrix_Sparse constructor.
+     *
+     * @param mat_size The size of one of the dimensions of the matrix (which is necessarily squared).
+     * @param cols_per_chunk An optional hyperparameter used to define the number of right-hand-sides to pass to Ginkgo solver calls. see default_cols_per_chunk.
+     * @param preconditionner_max_block_size An optional hyperparameter used to define the maximum size of a block used by the block-Jacobi preconditionner. see default_preconditionner_max_block_size.
```

Comment:
```suggestion
     * @param mat_size The size of one of the dimensions of the matrix (which is necessarily squared).
     * @param cols_per_chunk An optional hyperparameter used to define the number of right-hand-sides to pass to
     *                       Ginkgo solver calls. see default_cols_per_chunk.
     * @param preconditionner_max_block_size An optional hyperparameter used to define the maximum size of a block
     *                       used by the block-Jacobi preconditioner. See default_preconditionner_max_block_size.
```

There is only 1 n in preconditioner. The variable name is wrong

Reply:
Yes this is reported in #434 I will do the sed when all at once